src/components/ReportDetailed/index.js

```diff
@@ -37,50 +53,60 @@ class ReportDetailed extends Component<ReportDetailedType, ReportDetailedState>
   render() {
     const { round, roundAmount, projects } = this.props;
     const { opened } = this.state;
-
+    const { printable } = this.state; 
+    const {date, start, end, duration, project, notes} = printable; 
     return (
       <Accordion className="ReportDetailed">
         <Accordion.Title active={opened} onClick={this.toggleDetails}>
           <Icon name="dropdown" />
-          Detailed view
+          Detailed view - 
         </Accordion.Title>
         <Accordion.Content active={opened}>
+        <Message info>
+          <p>Select Columns to be in the print view</p>
+        </Message>
           <Table celled>
             <Table.Header>
               <Table.Row>
-                <Table.HeaderCell>
+                <Table.HeaderCell className={date ? null : 'no-print'}>
                   Date
+                  <Checkbox  toggle defaultChecked onClick={((e, data) => this.onChangePrintView(data, "date"))}/> 
```

Comment:
If you add `column="date"` to `<Checkbox />` it will be available in the `data` param of the callback.

This way you can have `onChange={this.onChangePrintView}`. Which makes a more readable and faster